src/js/main.js

```diff
@@ -274,17 +271,28 @@ var SupportKit = Marionette.Object.extend({
     },
 
     _checkConversationState: function() {
-
         if (!this._chatController.conversation || this._chatController.conversation.isNew()) {
             this._chatController._initConversation();
         }
     },
 
+    // Add placeholder to support back links to supportkit
+    _renderPlaceholder: function() {
+        $(_(function() {
+            if (!this.ready) {
+                var placeHolder = new ChatView().render();
+                placeHolder.main.show(new ConversationView());
+                this._placeHolder = $(placeHolder.el);
```

Comment:
I'd do 

```
this._placeHolder = new ChatView();
this._placeHolder.main.show(new ConversationView());
this._placeHolder.render().$el.addClass('placeHolder').appendTo('body');
```

in the current form, we are leaking a ChatView instance and a ConversationView instance because we only remove the element from the DOM and not disposing of those views.